src/test/java/org/mule/extension/db/integration/insert/InsertClobInputParamTestCase.java

```diff
@@ -36,6 +41,13 @@ public void usesStringOnExplicitParam() throws Exception {
     assertThat(response.getPayload().getValue(), equalTo(TEST_MESSAGE));
   }
 
+  @Test
+  public void usesStreamOnImplicitParam() throws Exception {
+    FileInputStream stream = new FileInputStream("src/test/resources/testFile.txt");
+    Message response = flowRunner("usesStringOnImplicitParam").withPayload(stream).run().getMessage();
+    assertThat(response.getPayload().getValue(), equalTo(FILE_TEST_TEXT));
+  }
+
```

Comment:
I did this test in my environment and it works fine without the new version of db-client? 
Could it be possible to generate the stream with the routine Guido used in the compression case and then compare the size of the parameter you provide to that routine with the field stored in the database?